client/src/components/Profiles.js

```diff
@@ -59,6 +109,8 @@ const Profiles = () => {
                 ></Table.Cell>
                 <Table.Cell children={profile.phone_number}></Table.Cell>
                 <Table.Cell children={profile.email}></Table.Cell>
+                <Table.Cell children={profile.occupation}></Table.Cell>
+                <Table.Cell children={profile.nationality_id}></Table.Cell>
```

Comment:
Don't show the `nationality_id` as a column. We don't need to see nationality in this view.